src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -187,43 +198,120 @@ export default class HaAutomationActionRow extends LitElement {
               .label=${this.hass.localize("ui.common.menu")}
               .path=${mdiDotsVertical}
             ></ha-icon-button>
-            <mwc-list-item>
+            <mwc-list-item
+              .label=${this.hass.localize(
```

Comment:
Same comment from #13457 applies here with the added caveat that `label` is not a valid property.  Let's get rid of these.

Reply:
@steverep I think your comment is based on an older commit.